reporters/reporter-default/src/main/java/org/radargun/reporting/html/TimelineDocument.java

```diff
@@ -37,6 +36,11 @@
    private Map<String, Integer> eventCategories = new TreeMap<String, Integer>();
    private long startTimestamp = Long.MAX_VALUE, endTimestamp = Long.MIN_VALUE;
 
+   @Override
+   public String getTitle() {
```

Comment:
I'd follow convention of having fields followed by constructor at first. Please move this e.g. below createTestCharts method.